src/cmake/FindNIFTYPET.cmake

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 University College London
+
+# This file is part of STIR.
+#
+# This file is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# See STIR/LICENSE.txt for details
+
+# cmake helper file, to be included by the root CMakeLists.txt
+# it sets 2 variables ALL_HEADERS ALL_INLINES
+# These are used for doxygen dependencies etc
+
+# The following macro finds all STIR headers and inlines.
+
+# The original MACRO was an adaptation from
+# https://cmake.org/pipermail/cmake/2012-June/050674.html
+# but we don't need the complicated stuff
+
+if (NOT NIFTYPET_ROOT_DIR)
+  set(NIFTYPET_ROOT_DIR "" CACHE PATH "Path to NIFTYPET")
+endif()
```

Comment:
not a great idea to have a name that ends on `_DIR` I guess (because that is used for "config" made), although not sure if there's a current CMake standard. Note that since recent CMake `*_ROOT` has special meaning